### ISSUE TITLE
TypeError: a class that defines __slots__ without defining __getstate__ cannot be pickled

### ISSUE THREAD

**Issue by scls19fr** [reclosedev/requests-cache#51]
Hello,

I get this strange error
`TypeError: a class that defines __slots__ without defining __getstate__ cannot be pickled`

    import requests
    import requests_cache

    filename_cache = "cache"
    requests_cache.install_cache(filename_cache, backend='sqlite', expire_after=60) # expiration seconds
    url = "http://www.google.fr"
    response = requests.get(url)
    print(response.text)

Any idea ?

Kind regards

**Comment by reclosedev** [comment]
Hello,
Which versions do you use?
```
python -c "import sys; import requests; import requests_cache; print(sys.version, requests.__version__, requests_cache.__version__)"
```
Can't reproduce on
```
3.4.0 (default, Apr 11 2014, 13:05:11) 
[GCC 4.8.2] 2.6.2 0.4.10
```

I had the same error but only with streaming requests for 2.6.2, it's fixed in 0.4.10. Can you try the latest version? 
Thanks.

**Comment by scls19fr** [comment]
Last version fix this issue. Sorry. Thanks.
